client/src/utils/windowSize.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
```

Comment:
@clubkemp didn't we add this also in profile redesign?

Reply:
yes we did, I needed it for this feature as well. They should be named and are the same exact thing so shouldn't be a merge conflict right?